packages/atomic/package.json

```diff
@@ -23,7 +23,7 @@
     "licenses/"
   ],
   "scripts": {
-    "build": "npm run build:locales && npm run build:stencil && npm run build:utils && npm run build:storybook",
+    "build": "npm run build:locales && npm run build:stencil && npm run build:utils && npm run doc:list-assets && npm run build:storybook",
```

Comment:
could we bundle the asset & storybook stuff in a separate script, and maybe keep it separate from `build`? I noticed a substantial increase in build time when debugging atomic because of it.

Reply:
Well, we want a single `build` command that is literally "build everything that's needed", not "most of it". AKA: That's the only command you need to run to build from scratch on CI.

If you need a separate script that only compiles some files or skip some steps when doing your day to day work, feel free to add them.